easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
The option is always there to make EB name the module any way you want, the question is just whether having it produce a module named `PGI/...` by default is OK...

In my view, lowercase module names (for specific modules, or all modules) is a site-specific policy.

The default policy we maintain in EB is to use the 'official' software name (i.e., the one most prevalent on the homepage), but it's not limiting in any way since we support custom module naming schemes.

I'm not sure which Lmod/Lua issue you're referring to. EasyBuild or Lmod mailing list?